ios/RNSentry.mm

```diff
@@ -207,6 +207,19 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
         }
     }];
 
+    NSDictionary<NSString *, id> *extraContext = [PrivateSentrySDKOnly getExtraContext];
+
+    NSMutableDictionary<NSString *, NSDictionary<NSString *, id> *> *deviceContext = [contexts[@"context"][@"device"] mutableCopy];
+    [deviceContext addEntriesFromDictionary:extraContext[@"device"]];
```

Comment:
M: What if `extraContext` is null? Or what if `extraContext` does not have `device`, does ObjC handle that automatically or would that throw?

Reply:
Currently, these two keys are always there but I agree it's not safe.

Reply:
Check https://github.com/getsentry/sentry-dart/pull/1402
I always check if there's a value and it's the type I expect.
https://en.wikipedia.org/wiki/Defensive_programming